app/models/story.rb

```diff
@@ -8,7 +8,7 @@ class Story < ActiveRecord::Base
   belongs_to :project, touch: true
   belongs_to :owner, class_name: "User"
   has_many :tasks, -> { order(completed: :asc, order: :asc, created_at: :asc) }
-  has_many :attachments
+  has_many :attachments, -> { where("attachable_id IS NOT NULL") }
```

Comment:
I would leave it the way it was, I found that out, but kinda liked that you could upload files without even posting the comment. Wouldn’t patch it until we are sure it needs patched, I don’t currently think that functionality is wrong.